tests/test_jointcal_decam.py

```diff
@@ -62,23 +62,23 @@ def test_jointcalTask_2_visits(self):
         # NOTE: The relative RMS limit was empirically determined from the
         # first run of jointcal on this data. We should always do better than
         # this in the future!
-        relative_error = 32e-3*u.arcsecond
+        relative_error = 19e-3*u.arcsecond
         pa1 = 0.14
         # NOTE: decam fits are currently not converging; the chi2 jumps around, so skip that Metric.
-        metrics = {'collected_astrometry_refStars': 8193,
-                   'collected_photometry_refStars': 8189,
-                   'selected_astrometry_refStars': 773,
-                   'selected_photometry_refStars': 773,
-                   'associated_astrometry_fittedStars': 8241,
-                   'associated_photometry_fittedStars': 8241,
-                   'selected_astrometry_fittedStars': 2261,
-                   'selected_photometry_fittedStars': 2261,
-                   'selected_astrometry_ccdImages': 17,
-                   'selected_photometry_ccdImages': 17,
-                   'astrometry_final_chi2': 2144.87,
-                   'astrometry_final_ndof': 4138,
-                   'photometry_final_chi2': 3809.24,
-                   'photometry_final_ndof': 2274,
+        metrics = {'collected_astrometry_refStars': 4866,
+                   'collected_photometry_refStars': 4865,
```

Comment:
Why such a dramatic reduction in the number of collected reference stars?

Reply:
I removed ccds 13 and 14 because there was something very strange about them that was making the resulting fit very weird. I'd known the decam test was weird, but I finally explored it a bit in this ticket, since I was using it as a ConstrainedAstrometryModel example. That resulted in a significant reduction in the size of the refstar search circle.